api/package.json

```
@@ -129,6 +130,7 @@
   "scripts": {
     "clean": "rm -rf node_modules",
     "cache:refresh": "node scripts/refresh-cache",
+    "coverage": "nyc npm test",
```

Compte tenu que ce script npm ne permet pas de détecter le code non utilisé, mais uniquement celui qui n'est pas exécuté par les tests, alors il serait préférable de le nommer `test:coverage`. Par ailleurs c'est un usage très répandu.

```suggestion
    "test:coverage": "nyc npm test",
```

---

Très juste. Je l'ai fait, en reprenant l'usage du `package.json du monorepo de préfixer par `local` si ça ne tourne pas en CI ou en production, juste sur le poste du développeur
`local:trigger-lint-on-commit`

---

Bien vu pour l'utilisation d'un préfixe `local` dans ce cas :+1: 

Par contre comme `test:coverage` est par ailleurs un usage répandu, je pense que `local:test:coverage` serait plus indiqué.

---

Ca me va, j'ai corrigé